jobs/webhooks/templates/config/bpm.yml.erb

```diff
@@ -1,6 +1,13 @@
 ---
+<%
+CONFIG_PATH = "/var/vcap/jobs/webhooks/config"
+%>
 processes:
 - name: webhooks
   executable: /var/vcap/packages/webhooks/bin/manager-webhooks
+  args:
+  - -stderrthreshold=INFO
+  env:
+    KUBECONFIG: <%= CONFIG_PATH %>/kubeconfig.yaml

```

Comment:
Why not use the kubeconfig of the apiserver job config?

Reply:
This job couldn't access the config path of the other job.
Is there some other way to share the files ?

Reply:
Sure.